portal-web/docroot/html/taglib/ui/page_iterator/lexicon/start.jsp

```diff
@@ -252,11 +254,11 @@ if (forcePost && (portletURL != null)) {
 							<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:;">...</a>
 
 							<div class="dropdown-menu dropdown-menu-top-center">
-								<ul class="inline-scroller link-list">
+								<ul class="inline-scroller link-list" data-current-index="<%= cur + 2 %>">
 					</c:if>
 
 					<%
-					for (int i = (cur + 2); i < pages; i++) {
+					for (int i = (cur + 2); i < ((cur + 2) + initialPages); i++) {
```

Comment:
Hey @dustinryerson 

It looks like in some cases this change will cause extra items to be displayed in the pagination. In this situation, there are 100 total pages, and when on page 97-99 it will display an extra 20 or so pages.

<img width="1111" alt="screen shot 2017-03-30 at 2 23 53 pm" src="https://cloud.githubusercontent.com/assets/2466445/24526878/a81f9e42-1554-11e7-80f4-74153ea21153.png">

Any ideas?